book/07-git-tools/sections/revision-selection.asc

```diff
@@ -180,6 +180,22 @@ Running `git show HEAD@{2.months.ago}` will show you the matching commit only if
 If you have a UNIX or Linux background, you can think of the reflog as Git's version of shell history, which emphasizes that what's there is clearly relevant only for you and your ``session'', and has nothing to do with anyone else who might be working on the same machine.
 ====
 
+[NOTE]
+.Escaping brackets in PowerShell
+====
+
+When using PowerShell on Windows, brackets like `{` and `}` are special characters and must be escaped.
+You can escape them with a backtick ` or put the commit reference in quotes:
+
+[source,console]
+----
+$ git show HEAD@{0}     # will NOT work on Windows
+$ git show HEAD@`{0`}   # OK
+$ git show "HEAD@{0}"   # OK
+----
+
+====
+
```

Comment:
```suggestion
When using PowerShell, brackets like `{` and `}` are special characters and must be escaped.
You can escape them with a backtick ` or put the commit reference in quotes:

[source,console]
----
$ git show HEAD@{0}     # will NOT work
$ git show HEAD@`{0`}   # OK
$ git show "HEAD@{0}"   # OK
----

====

```

I've just tested this on Linux. Using `git show HEAD@{0}` will not work on Linux either. It gives the following error:
`fatal: ambiguous argument 'HEAD@': unknown revision or path not in the working tree.`

Both "workarounds" work on Linux as well.